etc/telemetry/compose.yaml

```diff
@@ -38,3 +38,41 @@ services:
     ports:
       - "4317:4317"
     depends_on: [tempo]
+
+  # NOTE: Database part to avoid opening two more terminals for extra commands.
+  # Doesn't need to be in the diagrams, it's just for convenience.
+  migration:
+    image: ghcr.io/trustification/trustd:latest
+    # network_mode to avoid:
+    # migration-1   | Error: Connection Error: pool timed out while waiting for an open connection
+    # migration-1   | Caused by:
+    # migration-1   |   pool timed out while waiting for an open connection
+    network_mode: "host"
```

Comment:
Host `host` mode brings some disadvantages. It's better to run with the normal. The hostname would be `postgres` in this case, as that's the name of the service later on in the file.

Reply:
Thanks it worked!

```
postgres-1    | 2025-06-04 09:55:10.732 UTC [1] LOG:  database system is ready to accept connections
migration-1 exited with code 0
```